connector.py

```diff
@@ -0,0 +1,20 @@
+""" Connector """
+
+from connectors.core.connector import Connector, get_logger, ConnectorError
+from .operations import operations, _check_health
+logger = get_logger('cybereason')
+
+
+class Cybereason(Connector):
+    def execute(self, config, operation, params, **kwargs):
+        try:
+            params.update({"operation":operation})              
+            operation = operations.get(operation)
+            return operation(config, params)
+        except Exception as err:
+            logger.error('Cybereason:{}'.format(err))
```

Comment:
logger = get_logger('cybereason')
this would already add the name 'cybereason' to every log line

Reply:
will fix it in the next commit